Sources/AsyncKit/Docs.docc/index.md

```diff
@@ -0,0 +1,7 @@
+# ``AsyncKit``
+
+AsyncKit provides a number of extensions to both Swift's Concurrency primitives and NIO's futures to make working with them easier.
```

Comment:
Probably should leave a remark that the majority of this functionality is soft-deprecated at this point.